libimage/image.go

```diff
@@ -587,6 +587,9 @@ func (i *Image) Tag(name string) error {
 	return i.reload()
 }
 
+// to have some symmetry with the errors from containers/storage.
+var errTagUnknown = errors.New("tag not known")
```

Comment:
Should this be exported  in libpod/define?

Reply:
I don´t think we need to.  I cannot think of a scenario where users shouldn't error out.